packages/core/src/contracts/multisig/basemultisigproxy.ts

```diff
@@ -40,12 +52,24 @@ export class BaseMultisigProxy {
     return undefined;
   }
 
-  protected async resolveMultisig(): Promise<void> {
-    const owner = await this.instance.owner.callAsync();
+  /**
+   * Instantiate `this.multisig` if appropriate.
+   * @param multisigAddress (optional) If supplied, instantiate multisig from that, otherwise check if `this.contractOwner` is a multisig wallet and instantiate from that. Otherwise `this.multisig` remains undefined.
```

Comment:
Shouldn't `multisigAddress == contractOwner` in all situations? I think we can simplify this to just having `contractOwner` and verifying code there.

Otherwise we have a different owner from the multisig that we control, which is very unexpected logic

Reply:
You're right, I did it this way to support the previous logic, which supported *non* multisig newsrooms, where `contractOwner` would not be a multisig address, but just some random wallet.

It's maybe also confusing because in the case of e.g. `EIP20MultsigProxy` there isn't really a `contractOwner` - that's only relevant for `NewsroomMultisigProxy`.

Can we completely stop supporting non-multisig newsrooms? If so that makes cleaning this up easier.